source/less/components/additional.less

```
@@ -22,20 +46,42 @@
   font-weight: 400;
   text-transform: uppercase;
   color: #222222;
+
+  @media (min-width: @tablet-width) {
+    .font(20px, 20px);
+  }
 }
 
 .additional__cost {
   .font(14px, 14px);
   color: #444444;
 
+  display: flex;
+
   p {
     margin: 0;
   }
+
+  @media (min-width: @tablet-width) {
+    .font(16px, 20px);
+  }
+}
+
+.additional__button {
+  .font(16px, 20px);
+
+  @media (min-width: @tablet-width) {
+    grid-column: ~"1/-1";
```

такого формата я еще не видел. Можешь скинуть ссылку где так пишут. Я видел только так:
```suggestion
    grid-column: 1/-1;
```
наверное, это какое-то костыльное решение для less-препроцесора...

---

Плохих советов послушал. Спрашивал у других как реализовать такую функцию